engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -341,4 +341,37 @@ - (int64_t)lastAppModificationTime {
   return [fileDate timeIntervalSince1970];
 }
 
+- (UISceneConfiguration*)application:(UIApplication*)application
+    configurationForConnectingSceneSession:(UISceneSession*)connectingSceneSession
+                                   options:(UISceneConnectionOptions*)options {
+  // Not called before iOS 13.
+  if (@available(iOS 13.0, *)) {
+    NSDictionary* sceneManifest =
+        [[NSBundle mainBundle] objectForInfoDictionaryKey:@"UIApplicationSceneManifest"];
+    NSDictionary* sceneConfigs = sceneManifest[@"UISceneConfigurations"];
+    NSArray* roleConfigs = sceneConfigs[connectingSceneSession.role];
+
+    if (roleConfigs.count > 0) {
+      // Defer to the Info.plist if it is specified there.
+      return nil;
+    }
+
+    // This is the default storyboard in the flutter template.
+    UIStoryboard* storyboard = [UIStoryboard storyboardWithName:@"Main"
+                                                         bundle:[NSBundle mainBundle]];
+    if (!storyboard) {
+      return nil;
```

Comment:
if there's no storyboard, we should return the config with just `sceneClass` and `delegateClass` hooked up.

Reply:
Loading a storyboard that doesn't exist doesn't return nil either, that check will need to be changed.